src/inputs/date.html

```diff
@@ -1,21 +1,40 @@
 <script type="text/ng-template" id="wgn-config-input-date">
-<!-- UI Bootstrap Datepicker Popup directive https://angular-ui.github.io/bootstrap/ -->
+<!--
+	Zengine znDatetimepickerWrapper directive:
+	https://zenginehq.github.io/developers/plugins/api/directives/#zndatetimepickerwrapper
+
+	UI Bootstrap Datepicker Popup directive: https://angular-ui.github.io/bootstrap/
```

Comment:
the url is actually http://angular-ui.github.io/bootstrap/versioned-docs/0.12.1/#/datepicker

Reply:
Thanks. Added.